### PR TITLE
Add 404 Call To Action

### PR DESCRIPTION
**Pull request by AlexJBanks**
Prompt users who receive unexpected 404s to contact committee

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -7,3 +7,8 @@
 
 <h1 id="error-message"></h1>
 <ul id="link-list"></ul>
+<hr>
+<br>
+<br>
+<h4>Something wrong? Don't think you should have ended up here?</h4>
```
Comment (AlexJBanks) [inline 404.html:13]: Wording of this line is open to discussion

File: 404.html [hunk 2]
```diff
@@ -7,3 +7,8 @@
...
+<p>Send us an email at <a href=/email>cssoc@cs.bham.ac.uk</a> or reach out to one of <a href=/committee>committee</a>.</p>
```
Comment (Danyc0) [inline 404.html:14]: *the* committee

Comment (Danyc0) [reply 404.html:14]: Also, could you add an option for "contact one of the @committee on (the?) (CSS?) Discord"

**Review by Danyc0 — CHANGES_REQUESTED** [review]

File: 404.html (hunk 2 above)
Comment (AlexJBanks) [reply 404.html:14]: congrats on @-ing someone lmao

Comment (Danyc0) [reply 404.html:14]: oops

Comment (AlexJBanks) [reply 404.html:14]: I can add "the" easily, but there's no clean way to add a reference to @\Committee via discord easily.

Comment (AlexJBanks) [reply 404.html:14]: unless you want the @\Committee reference on the /committee page

Comment (Danyc0) [reply 404.html:14]: I don't mean to actually link to committee on discord, I mean literally just put the @ there to indicate that it's a discord role. A normal link to the discord can be added to the word "Discord" too

**Comment by AlexJBanks** [comment]
@Danyc0 are happy with the changes?

**Comment by JTWWilson** [comment]
Looks good as is but could add a pic TeX etc to make it fun

**Review by Danyc0 — APPROVED** [review]
Looks good to me, @JTWWilson can open a new PR to add a TeX if he wants, but I'll merge this one now

**Pull request merged**